tests/sps/validation/test_journal_meta.py

```diff
@@ -346,7 +346,7 @@ def test_validate_publisher_names_XML_has_surplus_items(self):
                 'response': 'ERROR',
                 'expected_value': ['Fundação Oswaldo Cruz'],
                 'got_value': ['Fundação Oswaldo Cruz', 'UNESP'],
-                'message': 'The following items are surplus in the XML: UNESP',
+                'message': 'The following items are not expected in the XML: UNESP',
```

Comment:
@Rossi-Luciano No lugar de are, coloque `is / are`

Reply:
OK, @robertatakenaka